N/NetCDF/NetCDF@julia-1.3/build_tarballs.jl

```diff
@@ -2,7 +2,6 @@ include("../common.jl")
 
 # Offset to add to the version number
 version_offset = v"0.1.0"
-
```

Comment:
This file shouldn't be touched.  You should instead touch [`N/NetCDF/NetCDF@julia-1.6/build_tarballs.jl`](https://github.com/JuliaPackaging/Yggdrasil/blob/0f5bbbe5800314419465747c07547a6ddb96bda5/N/NetCDF/NetCDF%40julia-1.6/build_tarballs.jl)

Reply:
Thanks.